_layouts/raw.html

```diff
@@ -28,7 +28,54 @@
 <script src="//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
 </head>
 <body class="mdl-layout mdl-js-layout mdl-layout--fixed-header mdl-layout--overlay-drawer-button">
-<header class=mdl-layout__header>
+    <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
+  <header class="mdl-layout__header">
+    <div class="mdl-layout__header-row">
+      <!-- Title -->
+      <span class="mdl-layout-title"><a href="{{site.baseurl}}/" class="mdl-layout-title mdl-navigation__link">IRkernel Documentation</a></span>
+      <!-- Add spacer, to align navigation to the right -->
+      <div class="mdl-layout-spacer"></div>
+      <!-- Navigation. We hide it in small screens. -->
+      <nav class="mdl-navigation mdl-layout--large-screen-only">
+        <a class="mdl-navigation__link" href={{site.baseurl}}/requirements/><i class="material-icons nav-icon">assessment</i>Requirements</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/installation/><i class="material-icons nav-icon">file_download</i>Installation</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/running/><i class="material-icons nav-icon">launch</i>Running</a>
+        <a class="mdl-navigation__link" href={{site.baseurl}}/faq/><i class="material-icons nav-icon">question_answer</i>FAQ</a>
```

Comment:
Just fixed this, here is an updated version of what the site looks like with the documentation link included:

![screen shot 2016-12-08 at 9 38 01 am](https://cloud.githubusercontent.com/assets/6437976/21020616/0ffc22ac-bd2a-11e6-805b-64b8395a5338.png)

![screen shot 2016-12-08 at 9 38 08 am](https://cloud.githubusercontent.com/assets/6437976/21020621/14271396-bd2a-11e6-892b-7166d6da396f.png)